pages/common/babeld.md

```diff
@@ -0,0 +1,24 @@
+# babeld
+
+> Routing daemon for Babel which uses firewall-style filters.
+> More information: <https://www.irif.fr/~jch/software/babel/babeld.html>.
+
+- Start babeld with a specific configuration file:
+
+`babeld -c {{babeld.conf}}`
```

Comment:
```suggestion
`babeld -c {{path/to/babeld.conf}}`
```

Could this be applied throughout with all files.

Reply:
Line 12 is then a bit long. Is that okay?

Reply:
Fine.